README.md

```diff
@@ -75,6 +75,7 @@ Please note that **Only a single AWS account that will be used for service billi
 
 To use this module with Terraform, follow these steps:
 
```

Comment:
Wondering whether for this particular part it would make sense to just link to the newly created documentation on that topic?